package.json

```diff
@@ -113,8 +113,8 @@
     "@emotion/cache": "^10.0.29",
     "@emotion/core": "^10.0.35",
     "@emotion/styled": "^10.0.27",
-    "@loadable/component": "~5.12.0",
-    "@loadable/server": "~5.12.0",
+    "@loadable/component": "~5.14.0",
+    "@loadable/server": "~5.14.0",
```

Comment:
is this just going introduce inconsistent builds? should we maybe keep it pinned but have something like this in the renovate config

```
    {
      "packagePatterns": ["^@loadable"],
      "enabled": false
    }
```

Reply:
I was confused by the lack of pinning there, I thought renovate would have pinned those? I haven't thought this through deeply but I think it may be impossible to truely pin everything as it can be very hard to find a combination of deps that works for everything without conflicts.